apps/web/src/views/Farms/v4/components/PoolListView.tsx

```diff
@@ -0,0 +1,202 @@
+  opacity: 0.1;
+  background: ${({ theme }) => theme.colors.contrast};
+`
+const ItemDetailFooter = styled.div`
+  border-top: 1px solid ${({ theme }) => theme.colors.cardBorder};
+`
+
+export const StyledButton = styled(Button)`
+  display: flex;
+  padding: 12px 16px;
+  justify-content: space-between;
+  gap: 8px;
+  align-self: stretch;
+  color: ${({ theme }) => theme.colors.text};
+  font-weight: 400;
+  line-height: 24px;
+`
+
+const ItemDetailBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 8px;
+  padding: 16px;
+  color: ${({ theme }) => theme.colors.textSubtle};
+  font-size: 12px;
+  font-weight: 600;
+`
+
+const ListItem = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: flex-end;
+  line-height: 150%;
+`
+
+const ListItemLabel = styled(Column)`
+  text-transform: uppercase;
+`
+
+export interface IListItemDetailsProps {
+  data: PoolInfo | null
+}
+
+const ListItemDetails: React.FC<IListItemDetailsProps> = ({ data }) => {
+  const { t } = useTranslation()
+  const columns = useColumnMobileConfig<PoolInfo>()
+
+  if (!data) {
+    return null
+  }
+
+  return (
+    <ItemDetailContainer>
+      <ItemDetailHeader>
+        <Grabber />
+      </ItemDetailHeader>
+      <ItemDetailBody>
+        <TokenOverview
+          isReady
+          token={data.token0}
+          quoteToken={data.token1}
+          width="48px"
+          icon={
+            <TokenPairImage
+              width={44}
+              height={44}
+              variant="inverted"
+              primaryToken={data.token0}
+              secondaryToken={data.token1}
+            />
+          }
+        />
+        {columns
+          .filter((col) => col.dataIndex !== null)
+          .map((col) => (
+            <ListItem key={col.key}>
+              <ListItemLabel>
+                {typeof col.title === 'function'
+                  ? col.title()
+                  : typeof col.title === 'string'
+                  ? col.title.toUpperCase()
+                  : col.title}
+              </ListItemLabel>
+              <Column>
+                {col.render
+                  ? col.render(col.dataIndex ? data[col.dataIndex] : data, data, 0)
+                  : col.dataIndex
+                  ? (data[col.dataIndex] as ReactNode)
+                  : null}
+              </Column>
+            </ListItem>
+          ))}
+      </ItemDetailBody>
+      <ItemDetailFooter>
+        <StyledButton scale="sm" variant="text" as="a">
+          {t('View pool details')}
+          <ChevronRightIcon />
+        </StyledButton>
+        <StyledButton scale="sm" variant="text" as="a">
+          {t('Add Liquidity')}
+          <ChevronRightIcon />
+        </StyledButton>
+        <StyledButton scale="sm" variant="text" as="a">
+          {t('View info page')}
+          <ChevronRightIcon />
+        </StyledButton>
+      </ItemDetailFooter>
```

Comment:
seems redundant to `PoolListItemAction`

Reply:
they are some diff in style, i can reuse them later when add the target link to them